cfp.html

```diff
@@ -16,7 +16,11 @@ <h1 class="c-red">
             <p>You’ll be joining over 16 years of tradition in inspiring and amazing Rubyists and other developers around Europe.</p>
             <p>A few things to note:</p>
             <ul>
+                <li>EuRuKo is a not-for-profit event.</li>
                 <li>All talks will have to adhere to our <a href="/coc">Code of Conduct</a>.</li>
+                <li>We are fully committed to have a lineup as diverse as we can.</li>
+                <li>As a not-for-profit event, we cannot offer to pay you for speaking. We can, however, cover travel and accommodation (see the perks section).</li>
+                <li>We will dedicate a few slots speakers that we will especially invite, including Matz.</li>
```

Comment:
How about: `We will dedicate a few slots for special speakers such as Matz.`

Reply:
That's much better, thank you!

Reply:
Not sure if a is needed before few, I think we don't need it. `We will dedicate few slots`

Reply:
Hmm, I think that changes the meaning.

How do you feel about `... a couple...`?

Reply:
Sounds right to me when singularity is used, for instance: `A slot will be dedicated for X.` versus `Few slots will be dedicated for X.` But I might be wrong since I am not a native speaker :) Let's leave it as it is.